addons/auth_signup/views/auth_signup_login_templates.xml

```diff
@@ -13,28 +13,43 @@
 
         <template id="auth_signup.fields" name="Auth Signup/ResetPassword form fields">
 
-            <div class="mb-3 field-login">
-                <label for="login">Your Email</label>
-                <input type="text" name="login" t-att-value="login" id="login" class="form-control form-control-sm" autofocus="autofocus"
-                    autocapitalize="off" required="required" t-att-readonly="'readonly' if only_passwords else None"/>
-            </div>
+            <t t-if="not is_from_reset_password">
```

Comment:
The template `auth_signup.fields` is being called from multiple places. To be specific, from `auth_signup.reset_password` and from `auth_signup.signup` as well.

Moving the code to the `auth_signup.signup` template would lead us to duplicating that code in the `auth_signup.reset_password` template.

However, in case I am missing something, I will be more than willing to correct myself.

Reply:
Great thought :+1: ,
Giving it a try right now...